docs/DeveloperGuide.adoc

```diff
@@ -12,9 +12,9 @@ ifdef::env-github[]
 :note-caption: :information_source:
 :warning-caption: :warning:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3/tree/master
+:repoURL: https://github.com/AY1920S1-CS2103T-F13-4/main.git
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `Team CS2103T-F13-4`      Since: `Sept 2019`      Licence: `NUS`
 
 == Setting up
 
```

Comment:
It seems like most of your URL diagrams are rather similar to the ones already given (except for occasional change in names). If you haven't updated your UML diagrams, please do so.